src/test/scala/mesosphere/marathon/api/akkahttp/DirectivesTest.scala

```diff
@@ -0,0 +1,83 @@
+package mesosphere.marathon
+package api.akkahttp
+
+import akka.http.scaladsl.model.StatusCodes.NotFound
+import akka.http.scaladsl.model.{ HttpEntity, StatusCodes, Uri }
+import akka.http.scaladsl.model.Uri.Path
+import akka.http.scaladsl.server.PathMatcher.{ Matched, Unmatched }
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import mesosphere.UnitTest
+import mesosphere.marathon.state.{ AppDefinition, PathId }
+import mesosphere.marathon.test.GroupCreation
+
+class DirectivesTest extends UnitTest with GroupCreation with ScalatestRouteTest {
+  import Directives._
+  import PathId.StringPathId
+
+  trait DirectivesTestFixture {
```

Comment:
I'm curious why this is a trait and not a class as we do it with other fixtures.